### PR TITLE
fix #58749 der-postillon.com

### LINKED ISSUE #58749: der-postillon.com
### Issue URL (Incorrect Blocking)
 [https://www.der-postillon.com/?fbclid=IwAR02bBqp8Et9cWOi4NjQ4JJiYFUMFNYhcRWjRZWO6wLEEVP9hM6pNgzGpVE](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.der-postillon.com%2F%3Ffbclid%3DIwAR02bBqp8Et9cWOi4NjQ4JJiYFUMFNYhcRWjRZWO6wLEEVP9hM6pNgzGpVE)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.0.1
Browser: | Safari
System wide filtering: | disabled
Simplified filters: | disabled
AdGuard DNS: | None
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard German

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#58749

### CODE REVIEW

File: SocialFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -2,6 +2,8 @@
 ! Extended CSS rules
 !
 underverse.su,underver.se#?##sidebar1-wrap > noindex:has(> h3:contains(Социальные сети))
+der-postillon.com##.sidebar > div.widget:has(> div.widget-content > a.twitter-timeline)
+der-postillon.com##.sidebar > div.widget:has(> div.widget-content > div#postillon24-videos > div.yt-subscribe-widget)
```
Comment (Alex-302) [inline SocialFilter/sections/css_extended.txt:6]: `##` => `#?#`

File: AnnoyancesFilter/sections/cookies_whitelist.txt [hunk 2]
```diff
@@ -6,6 +6,11 @@
 @@||quantcast.mgr.consensu.org/cmp.js$domain=cnews.fr
 @@||vendorlist.consensu.org/vendorlist.json$domain=cnews.fr
 @@||static.quantcast.mgr.consensu.org/*/cmpui-banner.js$domain=cnews.fr
+! https://github.com/AdguardTeam/AdguardFilters/issues/58749
+!#if (adguard_app_ios)
+@@||vendorlist.consensu.org/vendorlist.json$domain=der-postillon.com
+@@||marfeel.mgr.consensu.org/main.tcf.cmp.js$domain=der-postillon.com
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/cookies_whitelist.txt:12]: Are they not required for other platforms?

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_whitelist.txt (hunk 2 above)
Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/cookies_whitelist.txt:12]: I could not reproduce this problem on other platforms

File: SocialFilter/sections/css_extended.txt (hunk 1 above)
Comment (Sergey-Lyapin) [reply SocialFilter/sections/css_extended.txt]: fixed

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**